package-lock.json

```diff
@@ -0,0 +1,15253 @@
+{
```

Comment:
please delete this file, we are using yarn.lock to freeze our dependencies. Feel free to add it to the .gitignore file as well..

Reply:
Ok!